src/scenekit.cs

```diff
@@ -2329,22 +2331,22 @@ interface SCNSceneRenderer {
 	interface SCNSceneRendererDelegate {
 
 		[Export ("renderer:willRenderScene:atTime:")]
-		void WillRenderScene ([Protocolize]SCNSceneRenderer renderer, SCNScene scene, double timeInSeconds);
+		void WillRenderScene (ISCNSceneRenderer renderer, SCNScene scene, double timeInSeconds);
 
```

Comment:
That breaks classic and it seems the API is available on macOS (so we still care)